crates/ruff_linter/src/rules/pylint/rules/import_private_name.rs

```diff
@@ -48,7 +48,7 @@ use crate::checkers::ast::Checker;
 ///
 /// [PEP 8]: https://www.python.org/dev/peps/pep-0008/
 /// [PEP 420]: https://www.python.org/dev/peps/pep-0420/
-/// [`namespace-packages`]: https://beta.ruff.rs/docs/settings/#namespace-packages
+/// [namespace-packages]: https://beta.ruff.rs/docs/settings/#namespace-packages
```

Comment:
Do you mind removing this line entirely, and then reverting the `[namespace-packages]` additions? I thought options were special-cased such that they didn't need to be added as references (see, e.g., https://docs.astral.sh/ruff/rules/too-many-public-methods/, the settings linked in there aren't formatted as references in the source code). If it doesn't work though, we can merge as-is.

Reply:
After some playing around I found that this bit also needed stripping because it would mess up during generation otherwise (which could be why it was done like this in the first place?):
```diff
 /// ## Options
-/// - [`namespace-packages`][namespace-packages]: List of packages that are defined as namespace
-///   packages.
+/// - `namespace-packages`
```